docs/diagrams/CalendarSequenceDiagram.puml

```diff
@@ -0,0 +1,43 @@
+@startuml
+!include style.puml
+
+box UI UI_COLOR_T2
+participant ":MainWindow" as MainWindow UI_COLOR
+participant ":Calendar" as Calendar UI_COLOR
+participant "calendarList:ObservableList" as TimeEvents UI_COLOR
+end box
+
+participant "<<class>>\n:Collections" as Collections RANDOM_COLOR
+
+[-> MainWindow : executeCommand(command)
```

Comment:
i think the arguments need to specific (see other diagrams under Implementation)

Reply:
Oohh okaokay